addons/mail/views/mail_activity_views.xml

```diff
@@ -332,4 +344,85 @@
         </field>
     </record>
 
+    <record id="mail_activity_open_target_view_tree" model="ir.ui.view">
+        <field name="name">mail.activity.view.tree</field>
```

Comment:
:socks:
Name should include same details as xml id so that it is "understandable" in a list view of views (such views)
```suggestion
        <field name="name">mail.activity.view.tree.open.target</field>
```